contrib/intel/jenkins/tests.py

```diff
@@ -0,0 +1,309 @@
+import sys
+import os
+
+print(os.environ['CI_SITE_CONFIG'])
+sys.path.append(os.environ['CI_SITE_CONFIG']) # for adding path for ci_site_config
+
+import subprocess
+import re
+import ci_site_config
+import common
+import shlex
+
+class Test:
+    def __init__ (self, branchname, buildno, testname, core_prov, fabric,
+                  hosts, ofi_build_mode, util_prov=None):
+        self.branchname = branchname
+        self.buildno = buildno
+        self.testname = testname
+        self.core_prov = core_prov
+        self.util_prov = "ofi_{}".format(util_prov) if util_prov != None else "" 
+        self.fabric = fabric
+        self.hosts = hosts
+        self.ofi_build_mode = ofi_build_mode
+        if (len(hosts) == 2):
+            self.server = hosts[0]
+            self.client = hosts[1]
+       
+        self.nw_interface = ci_site_config.interface_map[self.fabric]
+        self.libfab_installpath = "{}/{}/{}/{}".format(ci_site_config.install_dir,
+                                  self.branchname, self.buildno, self.ofi_build_mode)
+ 
+        self.env = [("FI_VERBS_MR_CACHE_ENABLE", "1"),\
```

Comment:
This env variable has been deprecated from v1.8 onwards. If you plan to back-port Jenkinsfile to the v1.7.x, then you can add this there.

Reply:
aren't we planning to build/test against all branches right.

Reply:
wouldn't it be better to retain it than remove it. it may be harder to trace back the changes required later in case back porting is required? whereas at the moment it will work for both the cases.